docs/modules/release-notes/pages/0.26.adoc

```diff
@@ -19,51 +21,484 @@ To get started, follow xref:pkl-cli:index.adoc#installation[Installation].#
+This means that these users miss out on the benefits of project dependencies.
+
+Pkl 0.26 adds support for using dependencies in any hierarchical module scheme.
+This means that `modulepath:` users in Java can now declare projects and dependencies.
+This also means that other custom methods of embedding modules can also use dependency notation.
+
+[NOTE]
 ====
-XXX
+A _hierarchical_ URI is a URI whose path part starts with `/`, and in which `/` delimits path segments.
+
+The URI `myscheme:/foo.pkl` is hierarchical, whereas the URI `myscheme:foo.pkl` is not.
 ====
 
+To read more about this design, consult https://github.com/apple/pkl-evolution/tree/main/spices/SPICE-0005-scheme-agnostic-projects.adoc[SPICE-0005].
+
+[[typealias-scoping-fix-and-change]]
+=== Typealias scoping fix and change (https://github.com/apple/pkl/pull/144[#144], https://github.com/apple/pkl/pull/373[#373], https://github.com/apple/pkl/pull/516[#516])
+
+Currently, a typealias that references a value on its enclosing module will break if used in another module.
+
+.myModule.pkl
+[source,pkl]
+----
+typealias StartsWithFoo = String(startsWith(foo))
+
+foo = "foo"
+----
+
+[source,pkl]
+----
+import "myModule.pkl"
+
+myStr: StartsWithFoo = "fooey"
+----
+
+This results in error:
+
+[source]
+----
+–– Pkl Error ––
+Cannot find property `foo`.
+
+1 | typealias StartsWithFoo = String(startsWith(foo))
+----
+
+This is a language bug that has been fixed.
+
+In the process of fixing this bug, we realized that the rules around variable resolution within typealiases were problematic.
+Typealiases are meant to be statically defined, and shouldn't be able to see properties that can be late-bound.
+Thus, a new rule is introduced: a typealias can only reference properties/methods that are `const`.
+
+This is a breaking change. To read more about this breakage and remediation steps, reference <<typealiases-are-treated-as-const>>.
+
 == Noteworthy [small]#🎶#
 
 Ready when you need them.
 
-.XXX
-[%collapsible]
-====
-XXX
-====
+[[pkldoc-improvements]]
+=== Pkldoc improvements
+
+The following improvements have been made to xref:pkl-doc:index.adoc[Pkldoc]:
+
+1. The documentation for the standard library no longer shows dependent packages, because every package implicitly depends on the standard library (https://github.com/apple/pkl/pull/503[#503]).
+2. If the standard library exists in the package list, it is shown first (https://github.com/apple/pkl/pull/165[#165]).
+3. Fewer characters are percent-encoded. For example, the `(` and `)` characters are no longer percent-encoded (https://github.com/apple/pkl/pull/489[#489]).
+
+[[standard-library-changes]]
+=== Standard library changes (https://github.com/apple/pkl/pull/506[#506])
+
+Multiple changes have been more to the standard library.
+
+==== New property in `pkl.settings`
+
+Module `pkl.settings` has new property `http: Http?`.
```

Comment:
This section is meant to list all the changes made to the stdlib, so this doens't really seem out of place to me. But to help with the disjointed-ness, I'll add a section to this to link to the HTTP proxy section in the noteworthy. Also, changed this to bullet points instead of sub-sections.